doc/run.txt

```diff
@@ -1,13 +1,14 @@
 Synopsis
 
-    cordova-cli run [BUILD] [PROD] [TARGET] [EXP] [PLATS] [-- POPTS]
+    cordova-cli run [BUILD] [PROD] [TARGET] [EXP] [PLATS] [LIST] [-- POPTS]
 
-      BUILD:  --nobuild
-      PROD:   --debug|--release
-      TARGET: --device|--emulator|--target=FOO
-      EXP:    --experimental [EXPERIMENTALFLAGS]
-      PLATS:  PLATFORM [...]
-      POPTS:  platformopts
+      BUILD:       --nobuild
+      PROD:        --debug|--release
+      TARGET:      [DEVICECLASS]|--target=FOO
```

Comment:
drop the [] around DEVICECLASS.

[] means "optional", but TARGET was already optional above, so it doesn't make sense for DEVICECLASS to be optional too...